meta-balena-common/recipes-containers/balena-supervisor/balena-supervisor/update-balena-supervisor

```diff
@@ -12,6 +12,12 @@ Options:
   -h, --help
         Display this help and exit.
 
+  -i <SUPERVISOR IMAGE>, --supervisor-image <SUPERVISOR IMAGE>
+        Set supervisor image to update to. This can be an image path like:
+          registry2.balena-cloud.com/v2/8f3d658a373039a9c27b8f1811b0261d
+        or a friendly URL like:
+          <org>/<fleet>/<commit|version>
```

Comment:
```suggestion
          bh.cr/<org>/<fleet>/<commit|version>
```
We will have a few domains available for these vanity paths to work. The one above is the most simple.